server/src/main/kotlin/org/javacs/kt/Compiler.kt

```diff
@@ -72,6 +72,9 @@ import kotlin.script.experimental.jvm.JvmDependency
 import org.javacs.kt.util.KotlinLSException
 import org.javacs.kt.util.KotlinNullableNotNullManager
 import org.javacs.kt.util.LoggingMessageCollector
+import org.jetbrains.kotlin.config.*
+import java.util.*
```

Comment:
Please use concrete imports instead of wildcards.

Reply:
Fixed.